add prana integration documentation (#41779)
home-assistant/home-assistant.github.io: source/_integrations/prana.markdown

---
title: Prana recuperators

> description: Integration to control Prana recuperators fans, modes, and sensors.

Reviewer: ```suggestion
description: Integration to control Prana recuperators.
```

> ha_release: 2025.11.1

Reviewer: ```suggestion
ha_release: 2026.2
```

ha_iot_class: Local Polling
ha_codeowners:

>   - '@home-assistant/core'

Reviewer: This should be the code owner

ha_domain: prana
ha_integration_type: integration
related:
  - url: https://prana.ua
    title: Prana
---

> The **Prana recuperators** integration adds support for Prana recuperators to Home Assistant. You can control motors and operating modes, and read a range of sensors provided by the device.

Reviewer: ```suggestion
The **Prana recuperators** {% term integration %} lets you control your Prana recuperator. You can manage motors and their operating modes, and monitor a range of sensors provided by the device.
```

Use case: If you have a Prana recuperator and want to automate ventilation, monitor filter status, or integrate the recuperator with other Home Assistant automations, this integration helps you do that.

## Supported devices

- Devices with Wi‑Fi control and firmware version 47 or newer

## Unsupported devices

- Models without a local network interface

> - Devices with firmware older than 47

Reviewer: ```suggestion
- Devices with firmware version 46 or below
```

## Prerequisites

1. Connect the Prana recuperator to the same local network as Home Assistant.

## Configuration

This integration is installed via a **config flow** and uses mDNS to discover devices on your local network. Setup is two clicks: select the discovered device and confirm to add it.

### Discovery

This integration is discovered automatically over mDNS. There is no user input step. Select the discovered device and confirm to add it.

## Supported functionality

### Entities

The integration exposes the following entities.

> #### Fans / sliders

Reviewer: Only switches are merged in right now. You will need to open separate pull requests as other are preparted to merge.

---
title: Prana recuperators
description: Integration to control Prana recuperators.
ha_release: 2026.2
ha_iot_class: Local Polling
ha_codeowners:
  -'@prana-dev-official'
ha_domain: prana
ha_integration_type: integration
related:
  - url: https://prana.ua
    title: Prana
---

The **Prana recuperators** {% term integration %} lets you control your Prana recuperator. You can manage motors and their operating modes, and monitor a range of sensors provided by the device.

Use case: If you have a Prana recuperator and want to automate ventilation, monitor filter status, or integrate the recuperator with other Home Assistant automations, this integration helps you do that.

## Supported devices

- Devices with Wi‑Fi control and firmware version 47 or newer

## Unsupported devices

- Models without a local network interface
- Devices with firmware version 46 or below

## Prerequisites

1. Connect the Prana recuperator to the same local network as Home Assistant.

## Configuration

This integration is installed via a **config flow** and uses mDNS to discover devices on your local network. Setup is two clicks: select the discovered device and confirm to add it.

### Discovery

> This integration is discovered automatically over mDNS. There is no user input step. Select the discovered device and confirm to add it.

Reviewer 2: ```suggestion
{% include integrations/config_flow.md %}
```
We have a standard section describing how to add an integration. 
- https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L51
- https://developers.home-assistant.io/docs/documenting/integration-docs-examples#configuration

## Supported functionality

### Entities

The integration exposes the following entities.

#### Switches

- **Auto**
  - Description: Enable automatic control
- **Auto+**
  - Description: Enhanced automatic mode with quieter operation
- **Winter**
  - Description: Winter mode for defrosting behavior
- **Heater**
  - Description: Turn on heater (if available)
- **Bound**
  - Description: Bind or synchronize both fans and related parameters

## Data updates

The integration uses local polling. By default, Home Assistant polls the device every 10 seconds. If the device stops responding, entities are marked as *unavailable* until communication is restored.

## Known limitations

- No official support for some older Prana models
- Some indicators or device-specific details may only be available in the manufacturer's app

> - This integration does not provide cloud or remote control

Reviewer 2: ```suggestion
- No official support for some older Prana models.
- Some indicators or device-specific details may only be available in the manufacturer's app.
- This integration does not provide cloud or remote control.
```
[End sentences with a period](https://learn.microsoft.com/en-us/style-guide/punctuation/periods).

## Troubleshooting

### Device is not discovered

1. Make sure Home Assistant and the Prana device are on the same local network
2. Restart the device and Home Assistant

> 3. Check whether mDNS/LLMNR is allowed by your router

Reviewer 2: ```suggestion
1. Make sure Home Assistant and the Prana device are on the same local network.
2. Restart the device and Home Assistant.
3. Check whether mDNS/LLMNR is allowed by your router.
```

### Entities show as unavailable

- Check the device network connection

> - Ensure the device is powered on and reachable; entities become available automatically when communication is restored

Reviewer 2: ```suggestion
- Check the device network connection.
- Ensure the device is powered on and reachable; entities become available automatically when communication is restored.
```

## Community notes

- If you have a model that does not work as expected, add a note in the repository or community and include the model and firmware version

## Removing the integration

> This integration follows standard integration removal in Home Assistant. After you remove the integration from Home Assistant, we recommend checking the device settings in the Prana Online 2.0 app.

Reviewer 2: ```suggestion
This integration follows standard integration removal in Home Assistant. 

{% include integrations/remove_device_service.md %}

After you remove the integration from Home Assistant, we recommend checking the device settings in the Prana Online 2.0 app.
```

- **Bound**
  - Description: Bind or synchronize both fans and related parameters

## Actions

The Prana integration supports standard Home Assistant switch actions and can be used in automations and scripts.

### Supported actions

The following actions are supported for all Prana switch entities:

> - `switch.turn_on`

Reviewer: Only custom actions need to be documented. If there are no custom actions at this time, you don't need to have a section for this now.
